libs/ledgerjs/packages/cryptoassets/src/currencies.ts

```diff
@@ -2781,18 +2781,24 @@ export const cryptocurrenciesById: Record<CryptoCurrencyId, CryptoCurrency> = {
     managerAppName: "VeChain",
     ticker: "VET",
     scheme: "vechain",
-    color: "#00C2FF",
+    color: "#28008C",
     family: "vechain",
     units: [
       {
         name: "VET",
         code: "VET",
-        magnitude: 8,
+        magnitude: 18,
+      },
```

Comment:
Are there other unit values for VET? This might be fine for now.

Reply:
let's talk about this in the stand-up

Reply:
Checked with Neil/Vineet, they don't know if sub units equal ETHs. Neil will ask Peter/Tony and see if they can provide clarity. Imo, for now we are ok not defining sub units down to wei. If we agree, Ill add to Backlog to track into the future.